### ISSUE TITLE
Document class is unable to access XMLSecurity class

### ISSUE THREAD

**Issue by Yanchek99** [sportngin/saml_idp#39]
The call to create a SignedDocument is incorrect and needs to be prefixed with SamlIdp https://github.com/sportngin/saml_idp/blob/master/lib/saml_idp.rb#L78

```
def signed_document
   XMLSecurity::SignedDocument.new(to_xml)
end
```

to

```
def signed_document
   SamlIdp::XMLSecurity::SignedDocument.new(to_xml)
end
```

**Comment by jphenow** [comment]
Probably a load order issue if you experience bugs with it - I expect it to work under certain circumstances but that's an unacceptable behavior. 

PRs welcome that specify the whole constant.

**Comment by Yanchek99** [comment]
cool, if I can find sometime to fork and iterate I would be happy to send a PR!

**Comment by jphenow** [comment]
Believe this was fixed in #41
